bot/src/bot.js

```diff
@@ -28,6 +28,25 @@ function saveUpdate(db, author, channel, text) {
   });
 }
 
+const createDateRe = /create a report for ([0-9]{4}-[0-9]{2}-[0-9]{2})/;
```

Comment:
I think it looks great without the slash :)

    alfred: create a report for 2016-09-27